.github/workflows/flutter_test.yml

```diff
@@ -129,6 +129,18 @@ jobs:
         run: |
           flutter drive --driver=integration_test/test_driver/driver.dart --target=integration_test/sentry_widgets_flutter_binding_test.dart --profile -d "${{ steps.device.outputs.name }}"
 
+      - name: run native test
+        # We only have the native unit test package in the iOS xcodeproj at the moment.
+        # Should be OK because it will likely be removed after switching to FFI (see https://github.com/getsentry/sentry-dart/issues/1444).
+        if: ${{ matrix.target != 'macos' }}
```

Comment:
We could use `== iOS`, easier to read what's included than whats excluded.

Reply:
updated